CHANGELOG.md

```diff
@@ -1,4 +1,43 @@
+NOTES:
+* `datadog_monitor`: The silence attribute is beginning its deprecation process, please use `datadog_downtime` instead ([#221](https://github.com/terraform-providers/terraform-provider-datadog/issues/221))
+
+IMPROVEMENTS:
+* `datadog_monitor`: Use ForceNew when changing the Monitor type ([#236](https://github.com/terraform-providers/terraform-provider-datadog/issues/236))
+* Update the vendored go client to `v2.21.0`. ([#230](https://github.com/terraform-providers/terraform-provider-datadog/issues/230))
+* `datadog_synthetics_test`: Support syntetics monitors in composite monitors. ([#222](https://github.com/terraform-providers/terraform-provider-datadog/issues/222))
+* `datadog_downtime`: Add validation to tags, add timezone parameter, improve downtime id handling, add descriptions to fields. ([#204](https://github.com/terraform-providers/terraform-provider-datadog/issues/204))
+* `datadog_screenboard`: Add support for metadata alias in graphs. ([#215](https://github.com/terraform-providers/terraform-provider-datadog/issues/215))
+* `datadog_monitor`: Add default to `no data` timeframe of 10 minutes. ([#212](https://github.com/terraform-providers/terraform-provider-datadog/issues/212))
+* `datadog_screenboard`: Add `custom_bg_color` to graph config. ([#189](https://github.com/terraform-providers/terraform-provider-datadog/issues/189)) Thanks [milanvdm](https://github.com/milanvdm)
+
+BUGFIXES:
+* `datadog_timebaord`: Fix the `extra_col` from having a non empty plan when there are no changes. ([#231](https://github.com/terraform-providers/terraform-provider-datadog/issues/231))
+* `datadog_timebaord`: Fix the `precision` from having a non empty plan when there are no changes. ([#228](https://github.com/terraform-providers/terraform-provider-datadog/issues/228))
+* `datadog_monitor`: Fix the sorting of monitor tags that could lead to a non empty diff. ([#214](https://github.com/terraform-providers/terraform-provider-datadog/issues/214))
+* `datadog_monitor`: Properly save `query_config` as to avoid to an improper non empty diff. ([#209](https://github.com/terraform-providers/terraform-provider-datadog/issues/209))
+* `datadog_screenboard`: Change monitor schema to be of type String instead of Int. ([#154](https://github.com/terraform-providers/terraform-provider-datadog/issues/154)) Thanks [mnaboka](https://github.com/mnaboka)
+
+
+INTERNAL:
+* `datadog_monitor`: Start deprecating the silence attribute, support `-1` to unmute a scope, and get fix spruious diffs. ([#221](https://github.com/terraform-providers/terraform-provider-datadog/issues/221))
+* Fix up the GCP and User acceptance tests ([#238](https://github.com/terraform-providers/terraform-provider-datadog/issues/238))
+* Fix up the `datadog_downtime` and `datadog_monitor` acceptance tests. ([#234](https://github.com/terraform-providers/terraform-provider-datadog/issues/234))
+* Cleanup go mod. ([#227](https://github.com/terraform-providers/terraform-provider-datadog/issues/227))
```

Comment:
IMO this is not important at all for end users and I'd just leave it out of the changelog.